core/src/main/java/org/elasticsearch/index/query/FuzzyQueryParser.java

```diff
@@ -60,23 +68,23 @@ public FuzzyQueryBuilder fromXContent(QueryParseContext parseContext) throws IOE
                 if (token == XContentParser.Token.FIELD_NAME) {
                     currentFieldName = parser.currentName();
                 } else {
-                    if ("term".equals(currentFieldName)) {
+                    if (parseContext.parseFieldMatcher().match(currentFieldName, TERM_FIELD)) {
```

Comment:
nit: reading all this makes me think if we could get parseContext.parseFieldMatcher() once with a shorter local name at the beginning and then shorten the lines here a bit. Just for readability. I know at this points it's probably some tedious search/replace action, just throwing this in as a thought.